connector/connect/src/main/scala/org/apache/spark/sql/connect/dsl/package.scala

```diff
@@ -51,6 +51,22 @@ package object dsl {
             .build()
         ).build()
       }
+
+      def groupBy(
+        groupingExprs: proto.Expression*)(aggregateExprs: proto.Expression*): proto.Relation = {
+        val agg = proto.Aggregate.newBuilder()
+        agg.setInput(logicalPlan)
+
+        val groupingSet = proto.Aggregate.GroupingSet.newBuilder()
+        for (groupingExpr <- groupingExprs) {
+          groupingSet.addAggregateExpressions(groupingExpr)
```

Comment:
why do we add group by expression to aggregate expressions?

Reply:
renamed the proto to make this clear.